plotly/tools.py

```diff
@@ -807,7 +807,7 @@ def make_subplots(rows=1, cols=1,
 
     # Throw exception if non-valid kwarg is sent
     VALID_KWARGS = ['horizontal_spacing', 'vertical_spacing',
-                    'specs', 'insets', 'subplot_titles']
+                    'specs', 'insets', 'subplot_titles', 'column_width']
```

Comment:
Oh actually, you should add a docstring about this above.

Reply:
> Looks good to me

I also want to add a `row_width` param. I'll do that then ping you after your other comment is addressed